docs/config/README.md

```diff
@@ -446,13 +446,12 @@ it would have been `nixpkgs/pkgs`.
 
 ### Options
 
-| Variable            | Default       | Description                                                                      |
-| ------------------- | ------------- | -------------------------------------------------------------------------------- |
-| `truncation_length` | `3`           | The number of parent folders that the current directory should be truncated to.  |
-| `truncate_to_repo`  | `true`        | Whether or not to truncate to the root of the git repo that you're currently in. |
-| `prefix`            | `"in "`       | Prefix to display immediately before the directory.                              |
-| `style`             | `"bold cyan"` | The style for the module.                                                        |
-| `disabled`          | `false`       | Disables the `directory` module.                                                 |
+| Variable                    | Default                    | Description                                                                      |
+| --------------------------- | -------------------------- | -------------------------------------------------------------------------------- |
+| `truncation_length`         | `3`                        | The number of parent folders that the current directory should be truncated to.  |
+| `truncate_to_repo`          | `true`                     | Whether or not to truncate to the root of the git repo that you're currently in. |
+| `format`                    | `"[$path]($style) "` | The format for the module.                                                       |
```

Comment:
It seems that you missed a style option  here.

Reply:
Added it..